### PR TITLE
Update readme install instructions, add faq

### PR DESCRIPTION
**Pull request by ericschn**
My idea so far on updating the readme, still need more FAQs, and some other info as it looks like master readme is slightly different than beta

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -6,17 +6,45 @@ More information about the crc32 bypassing and remapping can be found in my '[By
 
 ![diablo 2 resurrected screenshot](https://github.com/ferib/D2R-Offline/blob/master/img/weird_group_flex.jpg?raw=true)
 
+# Setup
+
+1. You will need the Diablo II: Resurrected game files, you can use this tool to download them:
```
Comment (ferib) [inline readme.md:11]: Let's change this one to `You will need a copy of the 'Diablo II: Resurrected' game files.` to keep it legal, I know the tool is mentioned in the old readme but this is somewhat of a gray area and I won't like to go to much down that road.

File: readme.md [hunk 2]
```diff
@@ -6,17 +6,45 @@ More information about the crc32 bypassing and remapping can be found in my '[By
...
+Unzip and copy **both** `D2ROffline.exe` and `patches.txt` to your Diablo II: Resurrected game folder. They should be in the same directory as `Game.exe`.
+
+1. Double-click `D2ROffline.exe` to start the tool.  
+Please see the FAQ below if you have any issues.
+
+# Frequently Asked Questions
+
+**Q**: Why can't I resume playing a previously created character?  
+**A**: Once you quit the game after creating a new character, you will need to use the TCP\IP button and host a game with that character. Alternatively, you can use the -FixLocalSave argument with `D2ROffline.exe` to adjust your save files to allow offline play, see the Usage section below.
+
```
Comment (ferib) [inline readme.md:31]: This is looking good, maybe we should split this into a new file called`FAQ.md` and add a link too there?

File: readme.md (hunk 1 above)
Comment (ericschn) [reply readme.md:11]: Understood, so the readme should have no mention of the BNetInstaller?

Comment (ferib) [reply readme.md:11]: Yes, I don't want to support a tool that could be used to get unauthorized copies of the game, DMCA takedown's are a bitch.

File: readme.md (hunk 2 above)
Comment (ericschn) [reply readme.md:31]: Sounds good, I'll add a note about using TCP/IP to load an existing character in the installation instructions also.

File: readme.md (hunk 1 above)
Comment (ericschn) [reply readme.md:11]: Very true, you might want to remove the link to it from master now, this repo is already very popular.

**Pull request merged**